src/main/java/mate/academy/spring/model/request/CinemaHallRequestDto.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.model.request;
```

Comment:
```suggestion
package mate.academy.spring.dto.request;
```
check similar places in all project